website/docs/r/ses_identity_notification.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "aws"
+page_title: "AWS: ses_identity_notification"
+sidebar_current: "docs-aws-resource-ses-identity-notification"
+description: |-
+  Setting SES Identity Notification
+---
+
+# ses_identity_notification
+
+Setting SES Identity Notification for a domain
+
+## Example Usage
+
+```hcl
+resource "ses_identity_notification" "test" {
```

Comment:
Two notes:
* Missing `aws_` in resource type
* We should `terraform fmt` the configuration here to make it easier to read:

```hcl
resource "aws_ses_identity_notification" "test" {
  identity          = "${aws_ses_domain_identity.example.domain}"
  notification_type = "Bounce"
  topic_arn         = "${aws_sns_topic.example.arn}"
}
```